src/item.cpp

```diff
@@ -4728,6 +4848,11 @@ bool item::is_salvageable() const
     return !has_flag( "NO_SALVAGE" );
 }
 
+bool item::is_craft() const
+{
+    return typeId() == "craft";
```

Comment:
This is quite dangerous: imagine a modder who decides to name their precious item "craft". I'd compare against `making` being `nullptr` instead.

Reply:
Makes sense, thanks!